docs/versioning.rst

```diff
@@ -0,0 +1,90 @@
+.. _versioning:
+
+Vyper Versioning Guideline
+**************************
+
+Motivation
+==========
+
+Vyper has different groups that are considered "users":
+- Smart Contract Developers (using Vyper to write smart contracts)
```

Comment:
Lists need a blank line before and after in order to render properly

Reply:
```suggestion

- Smart Contract Developers (using Vyper to write smart contracts)
```